modules/odc-importing-codebase-from-git-to-create-application.adoc

```diff
@@ -51,7 +52,9 @@ The *Serverless Deployment* option is displayed in the *Import from git* form on
 
 . In the *Pipelines* section, select *Add Pipeline*, and then click *Show Pipeline Visualization* to see the pipeline for the application.
 
-. In the *Advanced Options* section, the *Create a route to the application* is selected by default so that you can access your application using a publicly available URL. You can clear the check box if you do not want to expose your application on a public route.
+. Optional: In the *Advanced Options* section, the *Target port* and the *Create a route to the application* is selected by default so that you can access your application using a publicly available URL.
++
+If your application does not expose its data on the default public port, 80, clear the check box, and set the target port number as the port number you want to expose.
```

Comment:
How does one set the target port number? By appending it to the URL, or is there a field where one must enter the port number?

Reply:
There is a field where one must enter the port number.